build-data/mappings-patch.tiny

```diff
@@ -3,3 +3,9 @@ tiny	2	0	spigot	mojang+yarn
 # Originally DistanceManager, which also implements DistanceManager, so clashes since the implemented class
 # is imported and not fully qualified. Easiest fix is to just change the name
 c	net/minecraft/server/level/PlayerChunkMap$a	net/minecraft/server/level/ChunkMap$ChunkDistanceManager
+
+# Craftbukkit changes the signature of methods in ComposterBlock
+c	net/minecraft/world/level/block/BlockComposter	net/minecraft/world/level/block/ComposterBlock
+	m	(Lnet/minecraft/world/level/block/state/IBlockData;Lnet/minecraft/world/level/World;Lnet/minecraft/core/BlockPosition;Lnet/minecraft/world/entity/Entity;)Lnet/minecraft/world/level/block/state/IBlockData;	d	extractProduce
+	m	(Lnet/minecraft/world/level/block/state/IBlockData;Lnet/minecraft/server/level/WorldServer;Lnet/minecraft/world/item/ItemStack;Lnet/minecraft/core/BlockPosition;Lnet/minecraft/world/entity/Entity;)Lnet/minecraft/world/level/block/state/IBlockData;	a	insertItem
+	m	(Lnet/minecraft/world/level/block/state/IBlockData;Lnet/minecraft/world/level/GeneratorAccess;Lnet/minecraft/core/BlockPosition;Lnet/minecraft/world/item/ItemStack;D)Lnet/minecraft/world/level/block/state/IBlockData;	a	addItem
```

Comment:
are these still needed? upstream mapped all the methods and changed the ones they changed.